lib/landlord.rb

```diff
@@ -148,5 +150,4 @@ def self.drop_schema(schema_name)
   def self.schema_exists?(name)
     ActiveRecord::Base.connection.schema_exists? name
   end
-  end
```

Comment:
removed the class. Seems much more appropriate to use `Landlord.switch_to Tenant.first` than `Landlord::Lanlord.switch_to Tenant.first`

Reply:
Fantastic